### PR TITLE
Erc721 labels

### PR DESCRIPTION
**Pull request by Yhtiyar**
## Changes
Added support of `erc721` labels which are added in #226

## How to test these changes?
manually

### CODE REVIEW

**Comment by Yhtiyar** [comment]
example request:
```
localhost:7481/address_info/ethereum_blockchain?address=0x1dfe7Ca09e99d10835Bf73044a23B73Fc20623DF
```

Response: 
```
{
  "address": "0x1dfe7Ca09e99d10835Bf73044a23B73Fc20623DF",
  "token": null,
  "smart_contract": null,
  "nft": {
    "name": "More Loot",
    "symbol": "MLOOT",
    "external_url": [
      "https://etherscan.io/token/0x1dfe7Ca09e99d10835Bf73044a23B73Fc20623DF",
      "https://www.blockchain.com/eth/address/0x1dfe7Ca09e99d10835Bf73044a23B73Fc20623DF"
    ],
    "total_supply": 76805
  }
}
```

File: backend/moonstream/actions.py [hunk 1]
```diff
@@ -107,6 +109,24 @@ def get_ethereum_address_info(
...
+        .one_or_none()
+    )
+    if erc721_label is not None:
+        address_info.nft = data.EthereumNFTDetails(
+            name=erc721_label.label_data.get("name"),
+            symbol=erc721_label.label_data.get("symbol"),
+            total_supply=erc721_label.label_data.get("totalSupply"),
+            external_url=[etherscan_token_url, blockchain_com_url],
+        )
+        breakpoint()
```
Comment (zomglings) [inline backend/moonstream/actions.py:129]: @Yhtiyar : Remove `breakpoint()`.

File: backend/moonstream/data.py [hunk 2]
```diff
@@ -160,18 +161,23 @@ class EthereumSmartContractSourceInfo(BaseModel):
...
+
+class EthereumNFTDetails(EthereumTokenDetails):
+    total_supply: Optional[int]
 
 
 class EthereumAddressInfo(BaseModel):
     address: str
     token: Optional[EthereumTokenDetails]
     smart_contract: Optional[EthereumSmartContractDetails]
+    nft: Optional[EthereumNFTDetails]
```
Comment (zomglings) [inline backend/moonstream/data.py:180]: Add `nft: Optional[EthereumNFTDetauls] = None`.

**Review by zomglings — APPROVED** [review]
lgtm

**Pull request merged**